lib/core/personUtils.js

```diff
@@ -0,0 +1,34 @@
+/**
+* Copyright (c) 2014, Tidepool Project
+*
+* This program is free software; you can redistribute it and/or modify it under
+* the terms of the associated License, which is identical to the BSD 2-Clause
+* License as published by the Open Source Initiative at opensource.org.
+*
+* This program is distributed in the hope that it will be useful, but WITHOUT
+* ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+* FOR A PARTICULAR PURPOSE. See the License for more details.
+*
+* You should have received a copy of the License along with this program; if
+* not, you can obtain one from Tidepool Project at tidepool.org.
+*/
+
+var _ = require('lodash');
+
+var personUtils = {};
+
+personUtils.patientFullName = function(person) {
```

Comment:
Did you copy this exactly from blip? Because the way we organize the user info in `allUsers` is not identical (apparently), and it doesn't work here... (I noticed because in the "home" side of the experience a "fake" child account shows up in the user dropdown with the parent's name instead of the patient name.)

A couple of unit tests would be good as well.